src/librustdoc/config.rs

```diff
@@ -619,6 +617,13 @@ impl Options {
         }
 
         let target = parse_target_triple(early_dcx, matches);
+        let maybe_sysroot = matches.opt_str("sysroot").map(PathBuf::from);
```

Comment:
Having a sysroot is mandatory in rustc, but not in rustdoc.
Can rustdoc do the same thing as rustc to make it mandatory?
(Then remove `Option` from `SearchPath::from_cli_opt`.)